src/map.js

```diff
@@ -1716,12 +1739,24 @@ var map = function (arg) {
   }
 
   /**
-   * Return the nearest valid bounds maintaining the
-   * width and height. Does nothing if m_clampBounds* is
-   * false.
+   * Return the nearest valid bounds maintaining the width and height.  Does
```

Comment:
It is easiest to show the issues that had existed and show what the new behavior allows.

For example of the current behavior, go here: http://opengeoscience.github.io/geojs/examples/tiles/?clampBoundsX=true&wrapX=false&zoom=0 using a browser with a window wider than it is tall.  This is our tile example with wrapping turned off and clamping turned on.  Hover your mouse over New Zealand, and use the mouse wheel to scroll in.  New Zealand will shift from underneath the cursor, so you zoom in on a different location.

This can be corrected by not clamping the bounds on zoom, but since clamping is a desired feature, there is code to clamp eventually (don't let the map get more than half off the screen), and to allow panning the map back to where the delta.  If you run the same example on this PR's code, New Zealand will stay under the mouse cursor.  You'll see white background to the right of New Zealand.  You can pan the map right (reducing that white space), but not left (since that increases the clamping violation).